### PR TITLE
de-humanize

### PR DESCRIPTION
**Pull request by ripienaar**
Signed-off-by: R.I.Pienaar <rip@devco.net>

### CODE REVIEW

File: server/jetstream.go [hunk 1]
```diff
@@ -474,7 +473,7 @@ func (a *Account) EnableJetStream(limits *JetStreamAccountLimits) error {
 		}
 
 		stats := mset.Stats()
-		s.Noticef("  Restored %s messages for MsgSet %q", humanize.Comma(int64(stats.Msgs)), fi.Name())
+		s.Noticef("  Restored %d messages for MsgSet %q", stats.Msgs, fi.Name())
```
Comment (derekcollison) [inline server/jetstream.go]: I liked the comma version here when messages get big. Can we snapshot that from humanize (giving credit of course)?

Comment (ripienaar) [reply server/jetstream.go]: I guess :) this even used to be a debug line, but sure let me see about writing some helper

Comment (derekcollison) [reply server/jetstream.go]: When I was doing my large tests, with 1B+ it was vert helpful.

Comment (ripienaar) [reply server/jetstream.go]: thats fair, I'll write a helper

Comment (ripienaar) [reply server/jetstream.go]: Not really sure what the right way is to give credit when pulling in a function like this let me know if something else is better - but there it is borrowed from humanize.

**Comment by derekcollison** [comment]
Looks like you have a go fmt issue which needs to be resolved.

**Comment by derekcollison** [comment]
Once that clears I will merge.

**Comment by ripienaar** [comment]
good now, sorry, new editor driving me crazy not doing fmt on save :)

**Pull request merged**